Resources/android/ExampleView.js

```diff
@@ -0,0 +1,20 @@
+var View = require('android.view.View'),
+  Activity = require('android.app.Activity'),
+  TiActivity = new Activity(Ti.Android.currentActivity);
+
+exports.create = function(params) {
+  params = params || {};
+
+  var view = new View(TiActivity);
+
+  if (params.backgroundColor) {
+
+    // Hyperloop should provide cross-platform helpers you can use anywhere in
+    // the app to do stuff like this:
+    var gColor = Hyperloop.colorFromString(params.backgroundColor);
+
+    view.setBackgroundColor(gColor);
+  }
```

Comment:
The problem is, that we need to make clear, if Hyperloop modules should be used 1:1 like the current modules, or the dev's need to take care themselves, if they want to expose stuff like `width` and `height` properties. 

Summarized: Because Hyperloop modules don't have a proxy, we a) need to provided those properties builtin or b) let the dev's decide.